src/mmw/sass/pages/_water-balance.scss

```diff
@@ -440,6 +440,38 @@
               background-color: $nlcd-90;
             }
           }
+
+          &.soil-a {
+            border-color: $soil-a;
+
+            &:after {
+              background-color: $soil-a;
```

Comment:
The white check-mark is not visible with these light colors:

![image](https://cloud.githubusercontent.com/assets/1430060/17118508/d9cb483a-528e-11e6-989f-5661afb54f8c.png)

Consider adding `color: #333` in the `&:after` block for all blocks that have dark text over a light background in the pop-up, namely High and Moderate Infiltration in the Soils block, and Wetlands, Grasslands, and Pasture / Hay in the Land Cover block.